### PR TITLE
[Doc] add resource tracking doc

### LINKED ISSUE #67: Doc: resource tracking
Add resource tracking doc to mkdoc

### PR DESCRIPTION
**Pull request by Sispheor**
Closes #67

### CODE REVIEW

**Comment by Sispheor** [comment]
I recommend to review this PR by checking out the code and executing mkdocs to get the final result
```
mkdocs serve -a 0.0.0.0:4000
```

File: docs/how_to/resource_tracking.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+## Link service catalog instances to resources
+
+Resources can be created from the API. It allows to create automatically a new resource in a resource group when 
+something is provisioned from the service catalog.
+
+In the example below, the playbook executed in Tower/AWX would have created a VM. 
+At the end of the process we call the squest API to instantiate a resource in the right resource group to reflect the 
+consumption.
+We link as well the pending instance(given by `squest.instance.id`) to this resource via the flag `service_catalog_instance`.
+```yaml
```
Comment (EliasBoulharts) [inline docs/how_to/resource_tracking.md:112]: Can you use the yaml file (playbook_examples/add_resource_in_resource_group.yml) instead of text?

Comment (Sispheor) [reply docs/how_to/resource_tracking.md:112]: I tried with the markdown extension "pymdownx.snippets" but the result is very bad

**Comment by Sispheor** [comment]
I have added the new logo has well and linked the favicon on it.

**Review by EliasBoulharts — APPROVED** [review]

**Review by a-belhadj — APPROVED** [review]

**Pull request merged**